src/Language/Haskell/LSP/VFS.hs

```diff
@@ -186,7 +186,11 @@ persistFileVFS vfs uri =
       let tfn = virtualFileName (vfsTempDir vfs) uri vf
           action = do
             exists <- doesFileExist tfn
-            unless exists (writeFile tfn (Rope.toString (_text vf)))
+            unless exists $ do
+               let contents = Rope.toString (_text vf)
+               logs  $ "haskell-lsp:persistFileVFS: Writing virtual file: " 
```

Comment:
Uses `logs` the default visibility? If so, it would clutter the output quite a lot in normal mode.

Reply:
The definitions is:
https://github.com/mpickering/haskell-lsp/blob/e709b32b6381aaa8274ae7cb0c2de938337187c6/src/Language/Haskell/LSP/Utility.hs#L49-L50

So i supposed it only will be shown in debug mode

Reply:
Right, didn't know that, thank you!